src/natcap/invest/coastal_vulnerability.py

```diff
@@ -2941,6 +2941,19 @@ def validate(args, limit_to=None):
                 (['shelf_contour_vector_path'],
                  'Must be a polyline vector'))
 
+    if ('slr_vector_path' in sufficient_keys and 
+       'slr_vector_path' not in invalid_keys):
+        vector = gdal.OpenEx(args['slr_vector_path'], gdal.OF_VECTOR)
+        layer = vector.GetLayer()
+        # all OGR point/multipoint codes
+        if layer.GetGeomType() not in [
+            ogr.wkbPoint, ogr.wkbMultiPoint,
+            ogr.wkbPointM, ogr.wkbMultiPointM,
+            ogr.wkbPointZM, ogr.wkbMultiPointZM,
+            ogr.wkbPoint25D, ogr.wkbMultiPoint25D]:

```

Comment:
Ahh there's always a surprising number of somewhat obscure geom types to handle. Using the constants like this is much nicer than hardcoding the integers like I did in the `shelf_contour_vector_path` test. 

What do you think about replacing those integers with the constants for that test as well? That would be a clear improvement for readability and make us more consistent across tests. I'll add an in-line comment in the test I'm referring to.

Reply:
Sure I think that would be an improvement!